tck/old-tck/source/install/jaspic/bin/ts.jte

```diff
@@ -520,6 +520,7 @@ impl.vi=glassfish
 impl.vi.deploy.dir=${s1as.domain}/autodeploy
 impl.deploy.timeout.multiplier=1200
 
+impl.ri=glassfish
```

Comment:
We don't consider GlassFish a reference implementation but historically we do of course.  We still have a lot of various references to `ri` things (in TCKs) but now they are just referencing any compatible implementation.

Reply:
@scottmarlow the changes in this PR are only about following the same structure as is used in the platform TCK to allow alignment of porting kits, the platform TCK uses RI so this matches the platform TCK for this specific ant target.

Reply:
Thanks for explaining Darran.  I'm fine with it for this pull request which introduces more compatibility for implementations that have previously run against the Platform TCK Authentication tests.

Going forward, if the old Authentication TCK tests will be refactored into the new Authentication TCK, that might be a better time to drop compatibility with the Platform TCK (legacy way).